src/uct/base/uct_iface.c

```diff
@@ -425,21 +425,26 @@ ucs_status_t uct_single_device_resource(uct_md_h md, const char *dev_name,
 ucs_status_t
 uct_base_iface_estimate_perf(uct_iface_h iface, uct_perf_attr_t *perf_attr)
 {
-    ucs_status_t status;
     uct_iface_attr_t iface_attr;
+    ucs_status_t status;
 
     status = uct_iface_query(iface, &iface_attr);
     if (status != UCS_OK) {
         return status;
     }
 
     /* By default, the performance is assumed to be the same for all operations */
+
```

Comment:
Blank line - intentionally?

Reply:
yes, since this comment applies to all 3 'if' statmements